src/App.jsx

```diff
@@ -14,10 +14,11 @@ import {
 } from '@mui/material';
 import React, { useEffect, useState } from 'react';
 import SplitPane from 'react-split-pane';
-import { bigIntReplacer, fetchChildren, fetchData, decodeValues } from './api';
+import { bigIntReplacer, decodeValues } from './api.js';
```

Comment:
Is the `.js` extension needed here? What about the other imports?

Reply:
This app builds without it but it's to help moving to ECMAScript Modules eventually.